frameworks/PHP/simps/app/Model/DbModel.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * This file is part of Simps.
+ *
+ * @link     https://simps.io
+ * @document https://doc.simps.io
+ * @license  https://github.com/simple-swoole/simps/blob/master/LICENSE
+ */
+
+namespace App\Model;
+
+use Simps\DB\BaseModel;
+
+class DbModel extends BaseModel
+{
+    public function fortunes()
+    {
+        $fortune = [];
+        $this->pdo->fortune_test = $this->pdo->fortune_test ?? $this->pdo->prepare('SELECT id, message FROM Fortune');
+        $this->pdo->fortune_test->execute();
+        $arr = $this->pdo->fortune_test->fetchAll();
+        foreach ($arr as $row) {
+            $fortune[$row['id']] = $row['message'];
+        }
+        $fortune[0] = 'Additional fortune added at request time.';
+        \asort($fortune);
```

Comment:
You are sorting an array, not the array of Fortunes objects.

Read the requiriments:
https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#requirements-3

Please update the code, or change the benchmark_config to raw db.

Reply:
All the db tests will use objects: db, updates and fortunes.

Reply:
Does this need to be revised again?

Reply:
No, thanks for the changes.
I liked the code for this fw.

Reply:
Thanks for the review.